src/components/main-form.js

```diff
@@ -559,7 +548,7 @@ class MainForm extends React.Component {
           </Card>
         </Col>
         <Col xs='8'>
-          <Summary className='p-4 mr-5'>
+          <div>
```

Comment:
Also, I removed the top margin, b/c if you merge in my navbar changes, it contains enough padding for the whole rest of the page